ArduPlane/Plane.h

```diff
@@ -385,8 +385,10 @@ class Plane : public AP_HAL::HAL::Callbacks {
     AP_BattMonitor battery;
 
 #if FRSKY_TELEM_ENABLED == ENABLED
+    // Inertial Navigation
+    AP_InertialNav_NavEKF inertial_nav {ahrs};
```

Comment:
do you want airspeed velocity or groundspeed velocity? if you want groundspeed, use ahrs.gps